examples/99-advanced/planets.py

```diff
@@ -0,0 +1,117 @@
+"""
+.. _planets_example:
+
+3D Earth and Celestial Bodies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example is inspired by `planet3D-MATLAB <https://github.com/tamaskis/planet3D-MATLAB>`_.
+
+
+Note that this example author doesn't know anything about astronomy. There may
+be inaccuracies in the representation. The purpose of this example is for
+inspiration only.
```

Comment:
Comment seemed a bit harsh to the author. I've reworded it.

```suggestion
.. note::
   The purpose of this example is to demonstrate plotting celestial bodies and
   may lack astronomical precision. There maybe inaccuracies in the
   representation, so please take care when reusing or repurposing this example.
   
   Please take a look at libraries like `astropy <https://www.astropy.org/>`_
   if you wish to use Python for astronomical calculations.
```